notebooks/week2/02_04_train_log_custom_model.py

```diff
@@ -0,0 +1,165 @@
+# Databricks notebook source
+import json
+from lightgbm import LGBMRegressor
+import mlflow
+from mlflow import MlflowClient
+from mlflow.models import infer_signature
+import numpy as np
+import pandas as pd
+from pyspark.sql import SparkSession
+from sklearn.pipeline import Pipeline
+from sklearn.metrics import mean_squared_error, mean_absolute_error, r2_score
+
+from power_consumption.config import ProjectConfig
+from power_consumption.utils import adjust_predictions
+
+mlflow.set_registry_uri("databricks-uc")
+mlflow.set_tracking_uri("databricks")
+client = MlflowClient()
+
+# COMMAND ----------
+
+try:
+    config = ProjectConfig.from_yaml(config_path="project_config.yml")
+except Exception:
+    config = ProjectConfig.from_yaml(config_path="../../project_config.yml")
```

Comment:
why do you need this logics? does not matter that much when we move into DABs, just curious

Reply:
I had issues with the path, depending on how I run the script. With this it is executable from the root path but also from the notebook path